reana_commons/config.py

```diff
@@ -461,7 +461,9 @@ def default_workspace():
 # Kerberos configurations
 
 KRB5_CONTAINER_IMAGE = os.getenv(
-    "KRB5_CONTAINER_IMAGE", "docker.io/reanahub/reana-auth-krb5:1.0.1"
+    # TODO: update to new version of image
```

Comment:
We can upgrade to 1.0.2.

Reply:
After https://github.com/reanahub/reana-auth-krb5/pull/19 is merged, we need to upgrade to 1.0.3